pom.xml

```diff
@@ -350,11 +350,11 @@
         </plugin>
         <plugin>
           <artifactId>maven-site-plugin</artifactId>
-          <version>3.4</version>
+          <version>3.6</version>
```

Comment:
This is the site plugin version used by the git plugin and the git client plugin and it has worked well for me.